<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta charset="UTF-8">
    <title>Data Stewardship Wizard</title>
    <link href="../_css/appendix.css" rel="stylesheet">
  </head>
  <body>
		<div class="public">
      <nav class="navbar navbar-expand-sm bg-primary fixed-top">
        <div class="container">
          <div class="navbar-header">
            <a class="navbar-brand" href="/">
              <img src="/img/logo.svg" alt="DSW logo">
              Data Stewardship Wizard
            </a>
          </div>
          <ul class="nav navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link" href="/questionnaires">Go to App</a>
            </li>
          </ul>
        </div>
      </nav>
      <div class="container">
        <div class="Public_BookReference">
          <div>
            <h1>Dataset documentation</h1>
            
            <h2>What's up?</h2>
            
            <p>
              To be interoperable, both file formats and data descriptions should 
			  follow established standards. To what extent will you use existing 
			  software that handle and produce results following standards? If you 
			  plan on developing tools for analysis yourself, will you use established 
			  file formats and data description standards (such as ontologies, 
			  structured controlled vocabularies)?
            </p>
            
            <h2>Do</h2>
			  <p>
			  Clearly note what format(s) your data will be in, see the table 
			  below for guidance:</p>
			  <table class="table-style1" style="width: 100%">
				  <tr class="table-style1">
					  <td class="table-th" style="width: 115px">Data 
					  type</td>
					  <td class="table-th" style="width: 126px">
					  Format name</td>
					  <td class="table-th" style="width: 870px">
					  Description</td>
					  <td class="table-th">URLs for repositories 
					  accepting data in this format</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" rowspan="3" style="width: 115px">
					  Raw DNA/RNA sequence</td>
					  <td class="table-tr1" style="width: 126px">FASTA FASTQ</td>
					  <td class="table-tr1" style="width: 870px">FASTA is a 
					  common text format to store DNA/RNA/Protein sequence and 
					  FASTQ combines base quality information with the 
					  nucleotide sequence. </td>
					  <td class="table-tr1">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" style="width: 126px">HDF5</td>
					  <td class="table-tr1" style="width: 870px">HDF5 is a 
					  newer sequence read formats used by long read sequencers 
					  e.g. PacBio and Oxford Nanopore. </td>
					  <td class="table-tr1">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" style="width: 126px">SAM BAM CRAM</td>
					  <td class="table-tr1" style="width: 870px">Raw sequence 
					  can also be stored in unaligned SAM/ BAM/ CRAM format.</td>
					  <td class="table-tr1">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" rowspan="3" style="width: 115px">
					  Assembled DNA sequence</td>
					  <td class="table-tr2" style="width: 126px">FASTA</td>
					  <td class="table-tr2" style="width: 870px">Assemblies 
					  without annotation are generally stored in FASTA format.
					  </td>
					  <td class="table-tr2">
					  http://www.ebi.ac.uk/ena/submit/genomes-sequence-submission</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" style="width: 126px">Flat file</td>
					  <td class="table-tr2" style="width: 870px">Annotation 
					  can be integrated with assemblies in contig, scaffold or 
					  chromosome flat file format. </td>
					  <td class="table-tr2">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" style="width: 126px">AGP</td>
					  <td class="table-tr2" style="width: 870px">AGP files are 
					  used to describe how smaller fragments are placed in an 
					  assembly but do not contain the sequence information 
					  themselves. </td>
					  <td class="table-tr2">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" style="width: 115px">Aligned DNA 
					  sequence</td>
					  <td class="table-tr1" style="width: 126px">SAM BAM CRAM</td>
					  <td class="table-tr1" style="width: 870px">Sequences 
					  aligned to a reference are represented in sequence 
					  alignment and mapping format (SAM). Its binary version is 
					  called BAM and further compression can be done using the 
					  CRAM format.</td>
					  <td class="table-tr1">
					  https://www.ncbi.nlm.nih. 
					  gov/sra/docs/submitformats/#bam</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" rowspan="3" style="width: 115px">
					  Gene model or genomic feature annotation</td>
					  <td class="table-tr2" style="width: 126px">
					  GTF GFF GFF3</td>
					  <td class="table-tr2" style="width: 870px">
					  General feature format or general transfer format are 
					  commonly used to store genomic features in tab-delimited 
					  flat text format. GFF3 is a more advanced version of the 
					  basic GFF that allows description of more complex 
					  features.</td>
					  <td class="table-tr2">
					  http://www.ensembl.org/info/website/upload/gff.html 
					  http://www.ensembl.org/info/website/upload/gff3.html
					  </td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" style="width: 126px">BED</td>
					  <td class="table-tr2" style="width: 870px">
					  BED format is a tab-delimited text format that also allows 
					  definition of how a feature should be displayed (e.g. on a 
					  genome browser)</td>
					  <td class="table-tr2">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" style="width: 126px">
					  GB GBK</td>
					  <td class="table-tr2" style="width: 870px">
					  GenBank flat file Format (GB/GBK) is also commonly used 
					  but not well standardised</td>
					  <td class="table-tr2">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" style="width: 115px">Gene 
					  functional annotation</td>
					  <td class="table-tr1" style="width: 126px">
					  GAF (GPAD and RDF will also be available in 2018)</td>
					  <td class="table-tr1" style="width: 870px">
					  A GAF file is a GO Annotation File containing annotations 
					  made to the GO by a contributing resource such as FlyBase 
					  or Pombase. However, the GAF standard is applicable 
					  outside of GO, e.g. using other ontologies such as PO. GAF 
					  (v2) is a simple tab-delimited file format with 17 columns 
					  to describe an entity (e.g. a protein), its annotation and 
					  some annotation metadata</td>
					  <td class="table-tr1">
					  http://geneontology.org/page/submitting-go-annotations</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" style="width: 115px">Genetic/ 
					  genomic variants</td>
					  <td class="table-tr2" style="width: 126px">VCF</td>
					  <td class="table-tr2" style="width: 870px">
					  A tab-delimited text format to store meta-information as 
					  header lines followed by information about variants 
					  position in the genome. The current version is VCF4.2</td>
					  <td class="table-tr2">
					  http://www.ensembl.org/info/website/upload/var.html</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" style="width: 115px">Interaction 
					  data</td>
					  <td class="table-tr1" style="width: 126px">PSI-MI XML 
					  MITAB</td>
					  <td class="table-tr1" style="width: 870px">
					  Data formats developed to exchange molecular interaction 
					  data, related metadata and fully describe molecule 
					  constructs.</td>
					  <td class="table-tr1">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" style="width: 115px">Raw 
					  metabolite profile</td>
					  <td class="table-tr2" style="width: 126px">mzML nmrML</td>
					  <td class="table-tr2" style="width: 870px">
					  XML based data formats that define mass spectrometry and 
					  nuclear magnetic resonance raw data in Metabolomics</td>
					  <td class="table-tr2">&nbsp;</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" style="width: 115px">Protein 
					  sequence</td>
					  <td class="table-tr1" style="width: 126px">FASTA</td>
					  <td class="table-tr1" style="width: 870px">
					  A text-based format for representing nucleotide sequences 
					  or protein sequences, in which nucleotides or amino acids 
					  are represented using single-letter codes.</td>
					  <td class="table-tr1">
					  https://www.uniprot.org</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr2" style="width: 115px">Raw proteome 
					  profile</td>
					  <td class="table-tr2" style="width: 126px">mzML</td>
					  <td class="table-tr2" style="width: 870px">
					  A formally defined XML format for representing mass 
					  spectrometry data. Files typically contain sequences of 
					  mass spectra, plus metadata about the experiment.</td>
					  <td class="table-tr2">
					  https://www.ebi.ac.uk/pride</td>
				  </tr>
				  <tr class="table-style1">
					  <td class="table-tr1" style="width: 115px">Organisms and 
					  specimens</td>
					  <td class="table-tr1" style="width: 126px">Darwin core</td>
					  <td class="table-tr1" style="width: 870px">
					  The Darwin Core (DwC) standard facilitates the exchange of 
					  information about the geographic location of organisms and 
					  associated collection specimens.</td>
					  <td class="table-tr1">&nbsp;</td>
				  </tr>
			  </table>
            
            <h2>Links</h2>
            
            <a href="https://doi.org/10.12688/f1000research.12344.2">Griffin PC, Khadake J, 
			LeMay KS et al. Best practice data life cycle approaches for the life sciences 
			[version 2; referees: 2 approved]. F1000Research 2018, 6:1618</a> <!-- Add url to page as href value -->
            
          </div>
        </div>
      </div>
		</div>
	</body>
</html>