<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Data Stewardship Wizard</title>
    <link href="../_css/appendix_prod.css" rel="stylesheet">
    <script src="./choose_css.js"></script>
  </head>
  <body>
		<div class="public">
      <nav class="navbar navbar-expand-sm bg-primary fixed-top">
        <div class="container">
          <div class="navbar-header">
            <a class="navbar-brand" href="/">
              <img src="/img/logo.svg" alt="SciLifeLab logo">
              Data Stewardship Wizard
            </a>
          </div>
          <ul class="nav navbar-nav ml-auto">
            <li class="nav-item">
              <a class="nav-link" href="/projects">Go to App</a>
            </li>
          </ul>
        </div>
      </nav>
      <div class="container">
        <div class="Public_BookReference">
          <div>
            <h1>MINSEQE</h1>
            <p>
                <a href="http://fged.org/projects/minseqe/">Minimal Information about a high-throughput Nucleotide 
                    SeQuencing Experiment</a><br>
                <h3>Sample annotation </h3>
                Essential sample annotation, including the experimental factors (e.g. “compound” and “dose”) 
                and their values.<br>
                Essential information about the biological system from which samples were taken must be 
                given, e.g. the organism, strain or cultivar (if known and if appropriate), the organism 
                part or tissue, and what treatment(s) was/were applied.
                <h3> Raw reads </h3>
                The set of read sequences and base=level quality scores for each assay, as generated by the 
                sequencing instrument, in a recognized format.
                <h3>Processed data</h3>
                Typically a tabular output
                <h3>Experiment and sample-data relationships</h3>
                Summary of the experiment and its goals, contact information, and any associated publication.<br> 
                Part of this description should be a table specifying sample=data relationships, i.e. which sample 
                has led to which raw data file or which data element in the processed data files. 
                <h3>Experimental and data processing protocols</h3>
                Description of how the nucleic acid samples were isolated, purified and processed prior to sequencing.<br>
                It is recommended that a summary of the instrumentation used, library preparation strategy, labelling 
                and amplification methodologies, alignment algorithms and data filtering used be provided. <br>
                Data processing and analysis protocols must be described in sufficient detail to enable unambiguous 
                data interpretation and to enable scientists to reproduce the analysis steps.  This should include, 
                but is not limited to, data rejection methods, data correction methods, alignment methods, data 
                smoothing and filtering methods and identifiers used for reference genomes to which the sequences 
                were mapped (when applicable).


            </p>                
          </div>
        </div>
      </div>
		</div>
	</body>
</html>